lib/src/metta/runner/stdlib.rs

```diff
@@ -1117,7 +1131,8 @@ pub fn register_common_tokens(metta: &Metta) {
     tref.register_token(regex(r"get-state"), move |_| { get_state_op.clone() });
 }
 
-pub fn register_runner_tokens(metta: &Metta, cwd: PathBuf) {
+//TODO: We should move search_paths into the execution environment, so they can be updated at runtime
```

Comment:
I don't quite follow your suggestion?  Are you saying we first embed an environment structure (containing search paths) in to the Metta object, and then the ImportOp ultimately accesses those?  That would definitely work.

Are you suggesting that?  or something else?

Reply:
That's a good idea to include the search paths inside the Metta object.  I'll make that change along with the other search path behavior changes below.

Reply:
Well doing this means that in case we have `a.metta` which imports `b.metta` then in `b.metta` `ImportOp` will search modules in a directory where `a.metta` is located. And I think it is not what is expected. I believe `b.metta` should not try to import modules from `a.metta`'s directory.

Reply:
If we're instantiating a new Metta object, it could have its own import paths.  Cloning a Tokenizer will have the effect of cloning the closure that instantiates the ImportOp, which currently owns the import paths and would own the Metta handle with this change.

But I am not totally clear on the desired behavior.

Reply:
I moved the search paths into Metta, because I think that just makes sense as a design.  The new behavior is identical to the old behavior with regard to the search path to use loading a module with its own runner (ie `load_module_space` and `new_loading_runner`)

Commit 5bf0420